extensions/amp-analytics/0.1/linker-reader.js

```diff
@@ -69,7 +66,10 @@ export class LinkerReader {
    * @return {?Object<string, string>}
    */
   parseAndCleanQueryString_(name) {
-    const parsedUrl = parseUrlDeprecated(this.win_.location.href);
+    const service = Services.urlForDoc(this.win_.document.documentElement);
```

Comment:
We might be able to remove this entirely and operate directly on the `location` object.

Reply:
@jridgewell yeah you are correct, this wasn't actually needed